site/content/docs/developing/custom-environment-resources.en.md

```diff
@@ -16,7 +17,36 @@ Which credentials would you like to use to create name? [profile default]
   > No, I'd like to import existing resources (VPC, subnets).
 ```
 
-When you select the default configuration, Copilot follows [AWS best practices](https://aws.amazon.com/blogs/containers/amazon-ecs-availability-best-practices/) and creates a VPC with two public and two private subnets, with one of each type in one of two Availability Zones. While this is a good configuration for most cases, Copilot allows some flexibility when you import your own resources. For example, you may bring a VPC with only two private subnets and no public subnets for your workloads that are not internet-facing. (For more details on the resources you'll need for isolated networks, go [here](https://github.com/aws/copilot-cli/discussions/2378).)
+You may use the import feature to bring a VPC with only two private subnets and no public subnets for your workloads that are not internet-facing. (For more details on the resources you'll need for isolated networks, go [here](https://github.com/aws/copilot-cli/discussions/2378).)
```

Comment:
Would this paragraph be better suited under the title "## Modifying Copilot's default resources"?

Reply:
oh I think the disabling public subnets is only possible with the "Import VPC" feature and not while adjusting our default settings, so I kept this as is. Let me know if I'm missing something!